xml/security_ldap_install.xml

```diff
@@ -30,9 +30,9 @@
   <para>
       There are two ways to configure and create a new instance: from a custom
       configuration file, and from an auto-generated template file.
-      You may use the auto-generated template without changes for a test instance,
+      You can use the auto-generated template without changes for a test instance,
       though for a production system you must carefully review it and make 
-      necessary changes.
+      the necessary changes.
```

Comment:
"**the** necessary" implies that there _will be_ changes. I have altered this to "any" to admit the possibility that  changes can be optional.